tests/urls.py

```diff
@@ -1,7 +1,10 @@
+from django.contrib import admin
 from django.urls import path
 
 from . import views
 
+admin.autodiscover()
```

Comment:
Issue: 

> Typically you won’t need to call this function directly as [AdminConfig](https://docs.djangoproject.com/en/5.0/ref/contrib/admin/#django.contrib.admin.apps.AdminConfig) calls it when Django starts.

https://docs.djangoproject.com/en/5.0/ref/contrib/admin/#django.contrib.admin.autodiscover

Reply:
Updated PR to remove it.